Samples/__LightCore.Integration.AspNetCore.Sample/Issues.md

```diff
@@ -0,0 +1,5 @@
+* Threading issue is related to the tests
+  * Test setup is done kinda wrong
+* ASP.NET Core Integration doesn't work because of an missing feature in LightCore
+  * We need to support dynamic mappings
```

Comment:
How should "dynamic mappings" look like? Is there some kind of spec for this?

Reply:
This is just an old description :) You remember our call? We thought that ASP.NET Core adds mappings after the IServericeProvider was created. But that was wrong, because the ServiceProvider is unmutable.

Reply:
Yes i do remember. You mean "immutable"? OK.

Reply:
exactly :-)